src/components/public-layout.tsx

```diff
@@ -0,0 +1,26 @@
+import { LogoIcon } from '@dhis2/ui'
+import type { FC } from 'react'
+import { Navigate, Outlet, useLocation } from 'react-router-dom'
+import styles from './public-layout.module.css'
+
+export const PublicLayout: FC = () => {
+    const location = useLocation()
+
+    if (location.pathname === '/') {
+        return <Navigate to="/sign-in" />
+    }
+
+    return (
+        <div className={styles.container}>
+            <div className={styles.nav}>
+                <h1 className={styles.header}>
+                    <LogoIcon className={styles.logo} />
+                    DHIS2 Playground
```

Comment:
Should this be "DHIS2 Playground"? (cc @tonsV2).
There are some orgs out there that use DHIS2 but have changed their logo, should this be done here too (incl. the displayed text?)?

Reply:
Will be done later, recorded in https://dhis2.atlassian.net/browse/DEVOPS-441